cluster/addons/dns/kube2sky/kube2sky_test.go

```diff
@@ -63,6 +64,20 @@ func (ec *fakeEtcdClient) RawGet(key string, sort, recursive bool) (*etcd.RawRes
 	return &etcd.RawResponse{StatusCode: http.StatusOK}, nil
 }
 
+func (ec *fakeEtcdClient) Get(key string) (string, bool) {
```

Comment:
We have RawGet() implemented already. Do we need an additional Get? If so, it should be a wrapper around RawGet().